src/state.js

```diff
@@ -34,15 +34,15 @@ export default {
   }, {
     "id": 5,
     "name": "Decolorized Iodine",
-    "description": "Injury of prostate",
+    "description": "Injury treatment",
     "reviews": 20,
     "rating": 1,
     "imgUrl": "http://dummyimage.com/120x245.jpg/cc0000/ffffff",
     "price": "$70.10"
   }, {
     "id": 6,
     "name": "Fresh Sugar Honey Tinted Lip Treatment SPF15",
-    "description": "Unspecified open wound of vagina and vulva",
+    "description": "Unspecified wound balm",
```

Comment:
Whoops. Sorry about that, I didn't pay attention to the data. I'll fix it for next project.